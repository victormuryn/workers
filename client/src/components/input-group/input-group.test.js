import React from 'react';
import renderer from 'react-test-renderer';
import InputGroup from './input-group';

describe(`InputGroup tests`, () => {
  it(`Input Group renders corrects`, () => {
    const tree = renderer.create(<InputGroup
      placeholder="Placeholder text"
      onChange={() => {}}
      label="Test group"
      required={false}
      minLength={1}
      value="test"
      name="test"
      type="text"
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
