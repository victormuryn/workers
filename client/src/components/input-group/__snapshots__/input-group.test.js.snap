// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputGroup tests Input Group renders corrects 1`] = `
<label
  className="auth-form__label"
>
  <span
    className="auth-form__label-text"
  >
    Test group
  </span>
  <div
    className="auth-page__input-group"
  >
    <div
      className="auth-page__input-underlined"
    >
      <input
        minLength={1}
        name="test"
        onChange={[Function]}
        required={false}
        type="text"
        value="test"
      />
      <span
        className="auth-page__input-label"
      >
        Placeholder text
      </span>
    </div>
  </div>
</label>
`;
